src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -137,30 +137,26 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
 
         #region Static initialization
 
-        public static void InitializeForTesting(List<TypeInfo> serializationProviders = null)
+        public static void InitializeForTesting(List<TypeInfo> serializationProviders = null, IExternalSerializer fallback = null)
         {
             try
             {
                 RegisterBuiltInSerializers();
                 BufferPool.InitGlobalBufferPool(new MessagingConfiguration(false));
                 RegisterSerializationProviders(serializationProviders);
                 AssemblyProcessor.Initialize();
-                fallbackSerializer = GetFallbackSerializer();
+                fallbackSerializer = fallback ?? GetFallbackSerializer();
             }
             catch (ReflectionTypeLoadException ex)
             {
                 throw ex.Flatten();
             }
         }
 
-        internal static void Initialize(List<TypeInfo> serializationProviders)
+        internal static void Initialize(List<TypeInfo> serializationProviders, IExternalSerializer fallback = null)
```

Comment:
I htink this is good - it'll allow us to keep any existing tests unchanged and won't affect anybody, unless they explicitly specify the IL one as the fallback.